Client/Frontend/Library/BookmarksPanel.swift

```diff
@@ -34,7 +34,13 @@ fileprivate class BookmarkFolderTableViewCell: TwoLineTableViewCell {
     override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
         super.init(style: style, reuseIdentifier: reuseIdentifier)
 
-        imageView?.image = UIImage(named: "bookmarkFolder")
+        // UITableViewCell.imageview isn't respecting image insets. The workaround is to assign a blank image to the image view (as with no image, the image view will not be drawn),
+        // and then add the image as a subview with a specific frame.
+        imageView?.image = UIImage.createWithColor(CGSize(width: 1, height: 1), color: .clear)
+        let nested = UIImageView(image: UIImage(imageLiteralResourceName: "bookmarkFolder"))
+        nested.frame = CGRect(x: 5, y: 5, width: 20, height: 20)
+        imageView?.addSubview(nested)
```

Comment:
I wonder if setting the contentMode to .center and changing the scale would work as well?
Something like this. This is usually how we deal with this iirc.
```
imageView?.image = UIImage(named: "bookmarkFolder")?.createScaled(CGSize(width: 20, height: 20))
imageView?.contentMode = .center
```

Reply:
That should work, I'll try it!

Reply:
Yup, replacing my patch with that,  thanks!